records/scrum_team_charter.md

```diff
@@ -0,0 +1,87 @@
+  - Reach out and be vulnerable
+
+- **Commit**
+  - Do what we say we will do (and communicate when things change)
+  - Keep practical details front of mind
+  - Prioritise team outcomes
+  - Be meaningfully wrong
+
+- **Collaborate**
+  - Teach, share information and decisions, learning early and often and recording this where people can find the information again
+  - Broadcast by default
+
+- **Engage**
+  - Engage in synchronous sessions
+  - Thumbs up/down in response to questions in a group
+  - Aim to ask questions first (curiosity, clean/clean-ish questions)
+  - Never hesitate to express ourselves
+
+## Things we never do
+
+- **During delivery**
+  - Land code without review
+  - Hide information or code
+  - Criticise the person instead of the code
+  - Go it alone when doing something outside of your comfort zone
+
+- **During communication**
+  - Make assumptions about others' behaviour (other than best intent)
+  - Bottle up frustrations without discussing them
+  - Raise voices to each other
+  - Be vague (share in a way others can't engage or disagree with)
+
+## When and how do we raise issues / ask for help?
+
+- **When**
+  - When we'd like help
+  - When we are ignorant and think someone else is better positioned to do something
+  - When we are overwhelmed or unsure
+  - When the exploration pauses or hits a dead end
+
+- **How**
+  - Raise during Daily Scrum/Retro/Teams thread
+  - Inbox/call someone if raising publicly isn't appropriate
+  - Share what we do know, what our opinion is, what we've already tried if applicable and point out what are unsure about/need help with.
+
+## How do we make decisions?
+
+- **Who**
+  - Product Owner decides prioritisation on backlog, and has final say on acceptance criteria (with a LOT of input from others)
+  - Developer working on a ticket makes implementation decisions not already clarified by rest of team, and seeks input where they need help
+  - Input is always sought from discipline experts (Product, Development, UX, QA)
+
+- **How**
+  - Empirical, consensus-based decision-making - enough evidence is presented to reach one
+  - Try things, inspect, gather feedback, adapt
+  - With respect/reference to the real restrictions/contraints we face
+  - With reference to product vision
+
+## What do we do when the rules are broken?
+
+- **Feedback and explore**
+  - Hold each other accountable. Call out behaviour that doesn't align to charter.
+  - Give clean feedback on what happened.
+  - Get curious about why it happened - ask clean(ish) questions.
+  - Expect introspection and accountability to be taken - and demonstrate that expectation through curiosity and give clean feedback
+
+- Raise with performance coaches if resolving at source hasn't worked.
+
+## How do we hold ourselves accountable?
+
+- Peer accountability
+- Admit our shortcomings
+- Don't be afraid to display vulnerability
+- Create a safe environment for transparency

```

Comment:
I don't link safety with transparency. I don't feel safe but I can be transparent and say that I don't feel safe.